doc/readme.md

```diff
@@ -0,0 +1,335 @@
+- Entity: Entitites sind die Elemente (Helden, Monster, Schatzkisten, etc.) des Spiels
+- Component: Components speichern die Datensätze der Entitäten (z.B. die Lebenspunkte)
+- System: Systeme beinhalten die eigentliche Logik und agieren auf die Components
+
+*Weiteres zum ECS im Dungeon erfahren Sie [hier](ecs/ecs_basics.md)*.
+
+Sie selbst nutzen und erweitern die Components und Systeme der Vorgaben. Sie werden ebenfalls neue Entities, Components und Systeme konzeptionieren und implementieren. So erschaffen Sie z.B. Ihre eigenen Monster und fallengespickte Level.
+
+Sie werden im Laufe der Praktika verschiedene Assets benötigen. Diese liegen per Default im `assets`-Verzeichnis. Sie können das Standardverzeichnis in der `build.gradle` anpassen.
+
+- Standardpfad für Texturen: `assets/`
+- Standardpfad für Charaktere: `assets/character/`
+- Standardpfad für Level-Texturen: `assets/textures/dungeon/`
+
+## Strukturen
+
+In diesem Abschnitt werden Ihnen die wichtigsten Klassen im Dungeon vorgestellt.
+
+![Struktur ECS](ecs/img/ecs.png)
+
+Game ist die Basisklasse von der alles ausgeht. Die Methode `Game#render` ist die Game-Loop. Die Klassen `Entity`, `Component` und `ECS_System` sind die Implementierungen des ECS.
+
+Die LevelAPI generiert, zeichnet und speichert das aktuelle Level. Mehr zum Thema Level erfahren Sie [hier](../level/readme.md).
+
+*Anmerkung: Das UML ist für bessere Lesbarkeit gekürzt.*
+
+*Anmerkung: Die in Gelb hinterlegten Klassen des UML-Diagramms sind für ein Basisverständnis des Dungeons nicht nötig.*
+
+- `entity` Die Java-Implementierung der Entitäten eines ECS
+- `component` Abstrakte Klasse, jedes Component im ECS leitet hiervon ab
+- `ECS-Systems` Abstrakte Klasse, jedes System im ECS leitet hiervon ab
+- LevelAPI: Kümmert sich darum, dass neue Level erzeugt und geladen werden
+- DungeonCamera: Ihr Auge in das Dungeon
+- libGDXSetup: Bereitet die Anwendung vor, für die Verwendung des Dungeons ist Verständnis für die genaue Funktionalität nicht notwendig
+- Game erstellt die Entitäten, Components und Systeme des ECS und beinhaltet die Game-Loop. Game ist Ihr Einstiegspunkt in das Dungeon
+- Game-Loop: Die Game-Loop ist die wichtigste Komponente des Spieles. Sie ist eine Endlosschleife, welche einmal pro Frame aufgerufen wird. Das Spiel läuft in 30 FPS (also 30 frames per seconds), die Game-Loop wird also 30-mal in der Sekunde aufgerufen. Alle Aktionen, die wiederholt ausgeführt werden müssen, wie zum Beispiel das Bewegen und Zeichnen von Figuren, müssen innerhalb der Game-Loop stattfinden. Das Framework ermöglicht es Ihnen, eigene Aktionen in die Game-Loop zu integrieren. Wie genau das geht, erfahren Sie im Laufe dieser Anleitung.
+*Hinweis: Die Game-Loop wird automatisch ausgeführt, Sie müssen sie nicht aktiv aufrufen.*
+
+Zusätzlich existieren noch eine Vielzahl an weiteren Hilfsklassen, mit denen Sie mal mehr oder mal weniger Kontakt haben werden.
+
+### Game
```

Comment:
Bin mir grade nicht sicher, aber ich glaube markdown kann auch Code im header, oder? 
Wenn ja, machen

Reply:
Ja, geht. Sieht aber extrem hässlich aus. 

In meinen Sachen bin ich dazu übergegangen, in den Headern keinen Inline-Code zu verwenden und stattdessen Italics zu nehmen. Ist aber nur meine Praxis, weiss nicht, wie andere das handhaben.

Reply:
@Lena241 Außerdem sollte das besser `### Klasse Game` heissen (ob nun mit Inline-Code oder mit Italics stattdessen).